web.py

```diff
@@ -19,7 +19,7 @@ def robots():
 
 @route('/track/<id>')
 def track(id):
-    details = Tracker().track(id)
+    details = Tracker(id).track(id)
```

Comment:
Don't like passing `id` in constructor as well as the track method. I think just passing it to track is cleaner.